diff --git a/enigma/src/main/java/cuchaz/enigma/translation/mapping/EntryRemapper.java b/enigma/src/main/java/cuchaz/enigma/translation/mapping/EntryRemapper.java
index ed5dba5..53f0105 100644
--- a/enigma/src/main/java/cuchaz/enigma/translation/mapping/EntryRemapper.java
+++ b/enigma/src/main/java/cuchaz/enigma/translation/mapping/EntryRemapper.java
@@ -63,7 +63,8 @@ public class EntryRemapper {
 			FieldEntry fieldEntry = (FieldEntry) obfuscatedEntry;
 			ClassEntry classEntry = fieldEntry.getParent();
 
-			mapRecordComponentGetter(vc, classEntry, fieldEntry, deobfMapping);
+			// Disable this because records with different field and method names are not supported yet. Plssss fix
+//			mapRecordComponentGetter(vc, classEntry, fieldEntry, deobfMapping);
 		}
 
 		boolean renaming = !Objects.equals(getDeobfMapping(obfuscatedEntry).targetName(), deobfMapping.targetName());
